framework/web/Request.php

```diff
@@ -582,6 +582,15 @@ public function getBodyParam($name, $defaultValue = null)
     {
         $params = $this->getBodyParams();
 
+        if (is_object($params)) {
+            // unable to use `ArrayHelper::getValue()` due to different dots in key logic and lack of exception handling
+            try {
+                return $params->{$name};
+            } catch (\Exception $e) {
+                return $defaultValue;
```

Comment:
Is that expected behavior?

Reply:
See https://github.com/yiisoft/yii2/blob/master/framework/helpers/BaseArrayHelper.php#L206-L209

I can not find any other way to handle unexisting object property here.

Reply:
OK.